docs/src/pages/docs/components/adapters.mdx

```diff
@@ -0,0 +1,10 @@
+---
+layout: docs
+category: Documentation
+permalink: /docs/components/adapters
+next: /docs/components/adapters
```

Comment:
these frontMatter props are from graphql.org, you can remove them

Reply:
Should we include them for SEO or any other related reason? Is it the whole block that we dont need?

Reply:
The whole block was used by previous graphql.org

Reply:
Got it, they are gone - thanks

* https://github.com/rocket-connect/graphql-debugger/pull/206/commits/2ea041b38f00487cbce39d308ca7c6ca46261e6b